ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -127,6 +131,16 @@ tests:
       ENABLE_HYPERSHIFT_CERT_ROTATION_SCALE: "true"
       REQUEST_SERVING_COMPONENT_TEST: "true"
     workflow: hypershift-aws-e2e-external
+- as: e2e-upgrade-tests-aws
```

Comment:
can we name this e2e-aws-upgrade-ho to be more consistent with the other names@openshift-ci
e2e-aks, e2e-aws, e2e-aws-4-17, e2e-kubevirt-aws-ovn-reduced...

Reply:
renamed it to `e2e-aws-upgrade-hypershift-operator`